dcpy/lifecycle/data_loader.py

```diff
@@ -24,7 +26,7 @@ def pull_dataset(ds: InputDataset, stage: str) -> Path:
     if ds.source is None:
         raise Exception(f"Cannot import a dataset with no source: {ds}")
 
-    ds.custom["file_type"] = ds.file_type
+    ds.custom["file_type"] = ds.file_type or "Other"
```

Comment:
this is a little kludgy... we need to revisit this later when we standardize things a little more around data_loader